templates/microstack/index.html

```diff
@@ -0,0 +1,358 @@
+          </p>
+          <p>
+            <a href="https://ubuntu.com/openstack/features">For large-scale deployments, try Charmed OpenStack&nbsp;&rsaquo;</a>
+          </p>
+        </div>
+        <div class="col-6 u-hide--small u-hide--medium u-align--center u-vertically-center">
+          {{
+          image (
+          url="https://assets.ubuntu.com/v1/3b176ef1-managed-private-cloud.svg",
+          alt="",
+          width="150",
+          height="150",
+          hi_def=True,
+          loading="auto"
+          ) | safe
+          }}
+        </div>
+      </div>
+      <hr class="is-fixed-width p-separator" />
+      <div class="row">
+        <div class="col-6 u-hide--small u-hide--medium u-align--center u-vertically-center">
+          {{
+          image (
+          url="https://assets.ubuntu.com/v1/74e1734e-edge-cloud.svg",
+          alt="",
+          width="250",
+          height="250",
+          hi_def=True,
+          loading="lazy"
+          ) | safe
+          }}
+        </div>
+        <div class="col-6">
+          <h2>OpenStack for cloud providers</h2>
+          <p class="p-heading--4">Open source equivalent to AWS Outposts and Azure Stack</p>
+          <p>
+            Cloud providers who operate an OpenStack public cloud usually need an extension of their infrastructure running on customer premises for privacy and data sovereignty reasons.
+          </p>
+          <p>
+            MicroStack answers these needs, delivering an out-of-the-box cloud solution with native OpenStack APIs that can be easily installed or shipped pre-installed to end users.
+          </p>
+          <p>
+            <a href="https://ubuntu.com/engage/public-cloud-openstack-implementation">Watch a webinar about public cloud implementation&nbsp;&rsaquo;</a>
+          </p>
+        </div>
+      </div>
+      <hr class="is-fixed-width p-separator" />
+      <div class="row">
+        <div class="col-6">
+          <h2>OpenStack for the edge</h2>
+          <p class="p-heading--4">Comprehensive cloud platform for distributed low-latency applications</p>
+          <p>
+            MicroStack answers the needs of multi-access edge computing (MEC), providing a secure, reliable and scalable cloud platform with a minimal footprint and simplified lifecycle management capabilities.
+          </p>
+          <p>
+            Ideal to meet requirements in telecom, industrial, automotive and other market sectors that need edge infrastructure.
+          </p>
+          <p>
+            <a href="https://ubuntu.com/engage/edge-infrastructure">Read a whitepaper about OpenStack on the edge&nbsp;&rsaquo;</a>
+          </p>
+        </div>
+        <div class="col-6 u-hide--small u-hide--medium u-align--center u-vertically-center">
+          {{ image (
```

Comment:
please remove space between the function name and the bracket and reformat the file with djLint